packages/design-system/src/components/mediaInput/index.js

```diff
@@ -88,7 +88,7 @@ const Img = styled.img`
 `;
 
 const menuStyleOverride = css`
-  min-width: 100px;
+  min-width: 115px;
```

Comment:
@swissspidy This fixes the width issue.

Reply:
Ah neat that we have an override here, makes it easy.

In that case, let's do this so that it also works for longer strings (e.g. when this is translated into German):

```suggestion
  white-space: nowrap;
```

![Screenshot 2022-07-01 at 14 09 17](https://user-images.githubusercontent.com/841956/176891675-2755186f-749c-44a1-8c78-521904e66cff.png)

Reply:
<img width="263" alt="Screenshot 2022-07-01 at 13 19 15" src="https://user-images.githubusercontent.com/237508/176893244-5ede7cae-ae1b-46dc-8d47-85690eb04454.png">

Good call. This is Brazilian.